test_mavros/include/sitl_test/test_type.h

```diff
@@ -0,0 +1,35 @@
+/**
+ * @brief SITL test type
+ * @file test_type.h
+ * @author Nuno Marques <n.marques21@hotmail.com>
+ *
+ * @addtogroup sitl_test
+ * @{
+ *  @brief SITL tests system
+ */
+/*
+ * Copyright 2015 Nuno Marques.
+ *
+ * This file is part of the mavros package and subject to the license terms
+ * in the top-level LICENSE file of the mavros repository.
+ * https://github.com/mavlink/mavros/tree/master/LICENSE.md
+ */
+
+#pragma once
+
+#include <tests/offboard_control.h>
+
+namespace testtype {
+class TestType {
```

Comment:
https://github.com/mavlink/mavros/pull/337/files#diff-a972179b5f960cd4f882fe71f48d887cR29. Haven't put one, but test cycle rates, number of test loops, setpoint precision before moving to the next one, etc.